app/api.py

```diff
@@ -37,19 +37,19 @@ def put_timestamp(hexhash: str, comment: str='', stream='timestamp') -> Optional
 
     client = get_active_rpc_client()
 
-    if comment:
-        data = dict(comment=comment)
-        serialized = ubjson.dumpb(data)
-        data_hex = hexlify(serialized).decode('utf-8')
-        response = client.publish(stream, hexhash, data_hex)
-    else:
-        response = client.publish(stream, hexhash)
-
-    if response['error'] is not None:
-        raise RpcResponseError(response['error']['message'])
-
-    return TxId(response['result'])
-
+    try:
```

Comment:
I wouldn´t recommend to wrap to many operations in one broad try/except. This code also blindly assumes that if there is an exception it must be an RpcResponseError. The could be different kinds of exceptions this code might throw. If you want  to log the exception here an re-raise it explicitly line 52 should contain the `raise` statment without hardcoding the exception type. But you might want to remove the complete try/except wrap as you do handle the exception anyways. The `log.debug(e)` is not required as exceptions will be handeled latest at: https://github.com/coblo/gui-demo/blob/develop/app/application.py#L29. As far as i know if app.is_frozen() is true (end-user installation without console) there is an automatic exception hook that shows the exception in a QT dialog. Also the mcrpc lib will raise its own `RpcError` Exception (not `RpcResponseError`).